app/uk/gov/hmrc/fhregistrationfrontend/controllers/BusinessPartnersPartnershipVatNumberController.scala

```diff
@@ -17,31 +17,40 @@
 package uk.gov.hmrc.fhregistrationfrontend.controllers
 
 import com.google.inject.{Inject, Singleton}
+import models.Mode
 import models.NormalMode
 import play.api.data.Form
 import play.api.mvc._
 import uk.gov.hmrc.fhregistrationfrontend.actions.Actions
 import uk.gov.hmrc.fhregistrationfrontend.config.FrontendAppConfig
 import uk.gov.hmrc.fhregistrationfrontend.forms.definitions.VatNumberForm.vatNumberForm
 import uk.gov.hmrc.fhregistrationfrontend.forms.models.VatNumber
+import uk.gov.hmrc.fhregistrationfrontend.pages.businessPartners.EnterVatNumberPage
+import uk.gov.hmrc.fhregistrationfrontend.repositories.SessionRepository
 import uk.gov.hmrc.fhregistrationfrontend.views.Views
 
+import scala.concurrent.{ExecutionContext, Future}
+
 @Singleton
 class BusinessPartnersPartnershipVatNumberController @Inject()(
   ds: CommonPlayDependencies,
   view: Views,
   actions: Actions,
-  config: FrontendAppConfig)(
+  config: FrontendAppConfig,
```

Comment:
FrontendAppConfig can be removed as it is unused

Reply:
Config is used below on line 53 to get the business type so this will need to stay.